ocaml/xapi-cli-server/cli_frontend.ml

```diff
@@ -1027,6 +1027,15 @@ let rec cmdtable_data : (string * cmd_spec) list =
       ; flags= [Host_selectors]
       }
     )
+  ; ( "host-apply-recommended-guidances"
```

Comment:
Small thing; while "guidances" is the plural of guidance, it is quite a rate word. I would have used the singular "guidance" (here and everywhere else) since a guidance may conceptually comprise multiple steps.

Reply:
There might be multiple recommended "guidances" after a update, i.e.: "RestartDeviceModel and RestartToolstack", and as the XAPI code from Centaurus was named this way, I would prefer to follow it in this PR. If we decide to change the naming, I think we can do it in a separate PR with low priority.